pkgs/servers/tmate-ssh-server/default.nix

```diff
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation rec {
   pname = "tmate-ssh-server";
-  version = "2.3.0";
+  version = "unstable-2021-10-17";
```

Comment:
Did you try to contact upstream to do a release BTW?

Reply:
Thanks for the hint. I sent a request to @nviennot who did the last release.